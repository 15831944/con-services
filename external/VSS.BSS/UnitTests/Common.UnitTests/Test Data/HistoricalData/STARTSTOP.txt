<?xml version="1.0" encoding="UTF-8"?>
<startStopTime:topic xmlns:startStopTime="http://www.cat.com/em/feed/v2/StartStopTime" xmlns:feed="http://www.cat.com/em/feed/v2/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v2/StartStopTime https://emfeed.cat.com/EMFeedFiles/schema/v2/StartStopTime.xsd">
<nextBuffer>
<url></url>
<moreData>false</moreData>
</nextBuffer>

   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315446</masterMsgId>
         <equipment>
            <serialNumber>-BYM00415</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200019512Q1</moduleCode>
         <moduleTime>2009-09-16 23:51:02</moduleTime>
         <receivedTime>2009-09-17 05:00:31</receivedTime>
      </header>
      <totalStartStops>5</totalStartStops>
      <startStopBlock>
         <startStop>
            <start>2009-09-16 06:44:56</start>
            <stop>2009-09-16 08:35:42</stop>
            <operatorId>0.0</operatorId>
         </startStop>
         <startStop>
            <start>2009-09-16 10:07:08</start>
            <stop>2009-09-16 10:07:18</stop>
            <operatorId>0.0</operatorId>
         </startStop>
         <startStop>
            <start>2009-09-16 10:32:46</start>
            <stop>2009-09-16 11:53:55</stop>
            <operatorId>0.0</operatorId>
         </startStop>
         <startStop>
            <start>2009-09-16 12:44:19</start>
            <stop>2009-09-16 14:23:12</stop>
            <operatorId>0.0</operatorId>
         </startStop>
         <startStop>
            <start>2009-09-16 15:06:31</start>
            <stop>2009-09-16 16:31:31</stop>
            <operatorId>0.0</operatorId>
         </startStop>
      </startStopBlock>
   </message>
   <message>
      <header>
         <messageId>0</messageId>
         <masterMsgId>119315586</masterMsgId>
         <equipment>
            <serialNumber>-A6G02420</serialNumber>
            <make>CAT</make>
         </equipment>
         <moduleCode>-DQ0200041542Q1</moduleCode>
         <moduleTime>2009-09-16 23:48:02</moduleTime>
         <receivedTime>2009-09-17 05:01:54</receivedTime>
      </header>
      <totalStartStops>3</totalStartStops>
      <startStopBlock>
         <startStop>
            <start>2009-09-16 10:14:12</start>
            <stop>2009-09-16 10:30:17</stop>
            <operatorId>0.0</operatorId>
         </startStop>
         <startStop>
            <start>2009-09-16 12:47:33</start>
            <stop>2009-09-16 12:56:47</stop>
            <operatorId>0.0</operatorId>
         </startStop>
         <startStop>
            <start>2009-09-16 13:59:52</start>
            <stop>2009-09-16 14:26:35</stop>
            <operatorId>0.0</operatorId>
         </startStop>
      </startStopBlock>
   </message>
</startStopTime:topic>
