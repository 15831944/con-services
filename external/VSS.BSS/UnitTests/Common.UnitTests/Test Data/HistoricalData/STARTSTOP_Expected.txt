<?xml version="1.0" encoding="utf-8"?><ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10"><NHDataWrapper><Data z:Id="i1" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T06:44:56</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i2" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T08:35:42</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i3" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T10:07:08</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i4" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T10:07:18</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i5" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T10:32:46</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i6" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T11:53:55</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i7" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T12:44:19</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i8" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T14:23:12</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i9" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T15:06:31</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i10" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>364</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T16:31:31</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315446</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i11" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>365</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T10:14:12</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315586</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i12" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>365</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T10:30:17</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315586</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i13" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>365</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T12:47:33</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315586</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i14" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>365</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T12:56:47</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315586</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i15" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>365</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T13:59:52</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>true</d3p1:IsStart><d3p1:SourceMsgID>119315586</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i16" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataEngineStartStop" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>365</d3p1:AssetID><d3p1:DebugRefID>0</d3p1:DebugRefID><d3p1:DeviceType>Unknown</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-09-16T14:26:35</d3p1:EventUTC><d3p1:GPSDeviceID i:nil="true" /><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsStart>false</d3p1:IsStart><d3p1:SourceMsgID>119315586</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper></ArrayOfNHDataWrapper>